### PR TITLE
Add patch to remove fd limit

### LINKED ISSUE #9: Pick up file descriptor patch?
### Comment:

@conda-forge/reproc How do you feel about picking up https://github.com/DaanDeMeyer/reproc/pull/103 in the feedstock to workaround the issue running mamba when the limit on the number of open file descriptors being more than 1024^2?

### PR DESCRIPTION
**Pull request by chrisburr**
Picks up the upstream patch for https://github.com/DaanDeMeyer/reproc/pull/103. As it hasn't been reviewed upstream I encourge someone to take a look at the patch first to see if there are any obvious problems as C isn't my forte.

Closes #9

### CODE REVIEW

**Comment by chrisburr** [comment]
@wolfv Here's a PR with the patch included which should be good to go, though:

> As it hasn't been reviewed upstream I encourge someone to take a look at the patch (https://github.com/DaanDeMeyer/reproc/pull/103) first to see if there are any obvious problems as C isn't my forte.

**Comment by chrisburr** [comment]
@wolfv Ping 😄

**Comment by chrisburr** [comment]
@wolfv Ping 😄

**Comment by AntoinePrv** [comment]
@chrisburr who you mind rebasing this? The feedstock structure has changed a bit.

**Comment by chrisburr** [comment]
@AntoinePrv Done!

**Comment by AntoinePrv** [comment]
I'm not a big fan of patching anything else than packaging issues in the feedstock, but it seems the upstream project is not so active...

Let's just add a comment in the recipe with the corresponding PR.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,9 +10,10 @@ source:
     sha256: 55c780f7faa5c8cabd83ebbb84b68e5e0e09732de70a129f6b3c801e905415dd
     patches:
       - apple_gettime.patch  # [osx]
+      - 103-remove-fd-limit.patch
```
Comment (chrisburr) [inline recipe/meta.yaml:17]:
```suggestion
      # When the open file descriptor limit > 1048576, reproc
      # (and therefore micromamba) aborts when starting a new process
      # As upstream is inactive, pick up the patch in the feedstock instead
      # https://github.com/DaanDeMeyer/reproc/pull/103
      - 103-remove-fd-limit.patch
```

**Review by AntoinePrv — APPROVED** [review]

**Pull request merged**